### ISSUE TITLE
Correct Format in Python.md

### ISSUE THREAD

**Issue by dipakkr** [dipakkr/A-to-Z-Resources-for-Students#19]
Correct formatting In pythton.md file

Use **database** bullet points for MySQL and mongo db

Use **testing** section and under that mention selenium resources

**Comment by shauryauppal** [comment]
Sorry, I know the resources but the way how it should look has to be corrected by you. 

I also know a lot of libraries that are really helpful in python but there is no section as such. Otherwise would have added.

**Comment by shauryauppal** [comment]
Yes once sections are added I will contribute. 
I know places links which have correct and good info but how to arrange please make the edits.

Selenium is Testing based.

MongoDB and Mysql is Database

Numpy and Pandas Library.

**Comment by dipakkr** [comment]
@shauryauppal  check I have updated the repo.

**Comment by dipakkr** [comment]
@shauryauppal any updates ??

**Comment by shauryauppal** [comment]
@dipakkr will do this now.

**Comment by Contrevien** [comment]
@shauryauppal Can you get those links in the Index (Python.md) to work? Thank you

**Comment by shauryauppal** [comment]
Links?

Links leading to where in Indexes?

**Comment by Contrevien** [comment]
Links leading to particular section in the Python.md document.

**Comment by shauryauppal** [comment]
Yes will check that how it is suppose to be done. I don't know how to hyperlink in Same .md file to different section.

There is contribution and now some things to learn from you guys.

**Comment by dipakkr** [comment]
@shauryauppal check the Table of content section in README file. It would be similar to that
